ecs-codedeploy

```diff
@@ -160,23 +160,24 @@ DEPLOY_WAIT_TIME=`aws deploy get-deployment-group \
     --query "deploymentGroupInfo.blueGreenDeploymentConfiguration.terminateBlueInstancesOnDeploymentSuccess.terminationWaitTimeInMinutes" \
     --output text`
 
-TOTAL_WAIT_TIME_MINUTES=$((DEPLOY_WAIT_TIME + 15))
-TOTAL_WAIT_TIME_SECONDS=$(((DEPLOY_WAIT_TIME + 15) * 60))
+TOTAL_WAIT_TIME_MINUTES=$((DEPLOY_WAIT_TIME))
```

Comment:
I'm pretty sure this will always fail without the added 15 minutes.
It takes 10 minutes for the traffic shifting + 30 minutes (in prod) before terminating the old task set (DEPLOY_WAIT_TIME) + 5 minute buffer time for whatever else.

Reply:
The script will sleep for 30 minutes (wait for termination). And the cmd below `aws deploy wait deployment-successful` will keep polling CodeDeploy status for 30 mins, which will wait until the deployment succeed https://docs.aws.amazon.com/cli/latest/reference/deploy/wait/deployment-successful.html

But we may want to test the events-api pipeline